rmf_door_msgs/msg/DoorRequest.msg

```diff
@@ -0,0 +1,7 @@
+builtin_interfaces/Time request_time
+string requester_id
+string door_name
+uint8 door_state
```

Comment:
Would it make sense to call this `door_mode` instead of `door_state` to differentiate it from `DoorState`?

Reply:
We could do something like

```
# DoorMode.msg
uint8 CLOSED=0
uint8 MOVING=1
uint8 OPEN=2
uint8 value
```

Reply:
I believe the enums get nested into the class, so in practice it would read as `DoorMode::CLOSED`, `DoorMode::OPEN`, etc.